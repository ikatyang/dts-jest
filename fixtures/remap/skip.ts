// @dts-jest:snapshot
Math.max(1, 2, 3);

// @dts-jest:skip:snapshot
Math.max(1, 2, 3);

// @dts-jest:snapshot
Math.max(1, 2, 3);
