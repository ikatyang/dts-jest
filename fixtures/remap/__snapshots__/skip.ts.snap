// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Math.max(1, 2, 3) 1`] = `"1"`;

exports[`Math.max(1, 2, 3) 2`] = `"2"`;

exports[`Math.max(1, 2, 3) 3`] = `"3"`;
